entrypoints/content/pages/settings/AppearanceForm.tsx

```diff
@@ -0,0 +1,154 @@
+// Packages:
+import { useEffect } from 'react'
+import useUserPreferences from '../../hooks/useUserPreferences'
+import { cn } from '../../lib/utils'
+
+// Typescript:
+import { Theme, Visibility } from 'types/user-preferences'
+
+// Components:
+import {
+  Select,
+  SelectContent,
+  SelectGroup,
+  SelectItem,
+  SelectTrigger,
+  SelectValue,
+} from '../../components/ui/select'
+
+// Functions:
+const AppearanceForm = () => {
+  // Constants:
+  const {
+    // TODO: Disable all inputs across /settings when isUserPreferencesLoading is true.
+    isLoading: isUserPreferencesLoading,
+    loadUserPreferences,
+    appearance,
+    setTheme,
+    setVisibility,
+  } = useUserPreferences()
+
+  // Effects:
+  useEffect(() => {
+    loadUserPreferences()
+  }, [])
+
+  // Return:
+  return (
+    <>
+      <div className='flex flex-row justify-between align-middle items-center'>
+        <div className='flex flex-col'>
+          <h3 className='text-sm font-base text-brand-primary'>Theme</h3>
+          <p className='text-xs font-base text-brand-secondary'>
+            Select the theme for the extension.
+          </p>
+        </div>
+        <Select
+          defaultValue={appearance.theme}
+          onValueChange={theme => setTheme(theme as Theme)}
+          disabled={!isUserPreferencesLoading}
+        >
+          <SelectTrigger className='text-xs w-44 h-fit px-2.5 py-2'>
+            <SelectValue placeholder='Select Theme' />
+          </SelectTrigger>
+          <SelectContent>
+            <SelectGroup>
+              <SelectItem value='Light'>Light</SelectItem>
+              <SelectItem value='Dark'>Dark</SelectItem>
+              <SelectItem value='System'>System Default</SelectItem>
+            </SelectGroup>
+          </SelectContent>
+        </Select>
+      </div>
+      <div className='flex flex-col justify-start align-middle items-start gap-4'>
+        <div className='flex flex-col'>
+          <h3 className='text-sm font-base text-brand-primary'>Visibility</h3>
+          <p className='text-xs font-base text-brand-secondary'>
+            Select how you’d like to launch OpenReply.
+          </p>
+        </div>
+        <div className='w-full h-fit flex flex-row flex-wrap gap-y-2 gap-x-4 justify-start'>
+          <div
+            className='flex flex-col gap-[2px] items-center cursor-pointer'
+            onClick={() => {
+              setVisibility(Visibility.ShowTopComment)
+            }}
```

Comment:
the three `onClick={() => setVisibility(Visibility)}` are not being disabled when `isUserPreferencesLoading` is true

we'll have to do the following for all the 4 visibility options below:
- `onClick={() => { if (!isUserPreferencesLoading) setVisibility(Visibility) }` i.e. we disable the click handler
- `cn('flex flex-col gap-[2px] items-center cursor-pointer', isUserPreferencesLoading && 'pointer-events-none cursor-default')` for this `div` only
- in the div that's rendering the mini-screen i.e. `'h-24 aspect-video bg-[#ECEDEF] rounded-sm relative'`, use cn and add a `isUserPreferencesLoading && opacity-85`